presto-docs/src/main/sphinx/develop/serialized-page.rst

```diff
@@ -177,8 +177,8 @@ MAP Encoding
 
 * Keys column
 * Values column
-* Hash table size - 4 bytes
-* [optional] Hash table
+* Hash table size (number of int32 in Hash Table) - 4 bytes
```

Comment:
Would it be clearer to say "number of 4-byte chunks in the hash table"?

Reply:
I can change that, although for me it sounds: "The color of it is green" VS "It is of green color". :)